pkg/cmd/experimental/syncgroups/ad/ldapinterface.go

```diff
@@ -73,15 +73,17 @@ func (e *ADLDAPInterface) ExtractMembers(ldapGroupUID string) ([]*ldap.Entry, er
 			return nil, err
 		}
 
-		for i := range currEntries {
-			currEntry := currEntries[i]
```

Comment:
Doesn't seem to break anything. I also don't understand why these two constructs would have different output - one is syntactic sugar for the other.